coderd/coderd.go

```diff
@@ -390,6 +390,7 @@ func New(options *Options) *API {
 		RedirectToLogin:             false,
 		DisableSessionExpiryRefresh: options.DeploymentValues.DisableSessionExpiryRefresh.Value(),
 		Optional:                    false,
+		SessionTokenFunc:            nil, // Default behavior
```

Comment:
Do we need to have the `// Default behavior` comment?

The comment on the field says: `If nil, the default behavior is used.`

Reply:
We don't need the comment, I was just being super explicit about what it means.